src/components/editor/actions/SaveAndPublishButton.jsx

```diff
@@ -27,7 +27,7 @@ const SaveAndPublishButton = (props) => {
   const showValidationErrors = () => dispatch(showValidationErrorsAction(resourceKey))
   const hideValidationErrors = () => dispatch(hideValidationErrorsAction(resourceKey))
 
-  const resourceHasChanged = useSelector(state => resourceHasChangesSinceLastSave(state))
+  const resourceHasChanged = useSelector(async state => await resourceHasChangesSinceLastSave(state))
```

Comment:
Looking at the related test failure, this is likely not doing what I had hoped and I'm not inclined to go down this rabbit hole.